api/http.go

```diff
@@ -13,8 +14,11 @@ import (
 
 	"github.com/gojekfarm/albatross-client-go/flags"
 	"github.com/gojekfarm/albatross-client-go/release"
+	"github.com/gorilla/schema"
 )
 
+var encoder = schema.NewEncoder()
+
 // APIClient defines the contract for the http client implementation to send requests to
 // the albatross api server
 type APIClient interface {
```

Comment:
we need to change this interface to adhere with HTTP client [Do](https://golang.org/pkg/net/http/#Client.Do) method eventually.